frontend/app/src/components/home/BalanceWithRefresh.svelte

```diff
@@ -29,12 +29,6 @@
             ? client.formatTokens(value, tokenDetails.decimals)
             : convertValue(conversion, tokenDetails);
 
-    $: {
-        if (ledger) {
-            refresh();
```

Comment:
I'm really not sure why this causes refresh to run multiple times given that the component's key is `ledger`... but it does.
With the call to `refresh` outside of this reactive statement the refresh function only ever triggers once.